lib/auth/crownjewel/crownjewelv1/service_test.go

```diff
@@ -180,7 +180,7 @@ type fakeChecker struct {
 
 func (f fakeChecker) CheckAccessToRule(_ services.RuleContext, _ string, resource string, verb string) error {
 	if resource == types.KindCrownJewel {
-		for slices.Contains(f.allowedVerbs, verb) {
+		if slices.Contains(f.allowedVerbs, verb) {
```

Comment:
How did you end up finding this? Linter?

Reply:
I copied the test and noticed it while making edits, in practice it makes no difference